tailwind.config.js

```diff
@@ -20,7 +23,10 @@ module.exports = {
       colors: {
         primary: "#165C9C",
         secondary: "#101828",
-        gray: "#667085"
+        gray: "#667085",
```

Comment:
Why not use tailwind colors instead?

Reply:
yea don't know the particular color in tailwind colors

that was why i defined them there